climpred/tests/test_compute_dims.py

```diff
@@ -98,10 +98,11 @@ def test_bootstrap_pm_dim(perfectModelEnsemble_initialized_control):
         comparison="m2c",
         iterations=ITERATIONS,
         resample_dim="member",
+        reference=["uninitialized"],
     )["tos"]
     assert "init" in actual.dims
-    for kind in ["initialized", "uninitialized"]:
-        actualk = actual.sel(kind=kind, results="skill").isnull().any()
+    for skill in ["initialized", "uninitialized"]:
+        actualk = actual.sel(skill=skill, results="verify skill").isnull().any()
```

Comment:
Is this a typo? Should it be `actual = actual.sel(...)`?

Reply:
no I want to check over the loop